cmk/special_agents/agent_cisco_meraki.py

```diff
@@ -5,14 +5,14 @@
 
 from __future__ import annotations
 
-import abc
 import argparse
 import logging
+from abc import abstractmethod
 from collections.abc import Iterable, Iterator, Mapping, Sequence
 from dataclasses import dataclass
 from enum import auto, Enum
 from pathlib import Path
-from typing import Final
+from typing import Any, Final
```

Comment:
Avoid "Any": from a typing perspective all consumer actions are OK/allowed (which is very bad).
You can use object instead.